### ISSUE TITLE
Approve pull requests

### ISSUE THREAD

**Issue by dentarg** [dennisreimann/ioctocat#467]
Not sure if GitHub API includes this yet, but it would be great to be able to approve pull requests via the app.

**Comment by dennisreimann** [comment]
You mean to merge pull requests? That's already offered by the Basic upgrade option.

**Comment by dentarg** [comment]
@dennisreimann nope, I want to [approve](https://github.com/blog/2256-a-whole-new-github-universe-announcing-new-tools-forums-and-features) pull requests

I checked, it isn't available yet, not even in the early access: https://developer.github.com/early-access/platform-roadmap/ ("Pull Request Review REST API")

**Comment by dennisreimann** [comment]
Thanks for the link and the clarification! 👍🏻

**Comment by rktoomey** [comment]
https://developer.github.com/v3/pulls/reviews/

Hi, this feature is now available via the API and my team has switched to using it. It would make this app a lot more useful if I could still use it to review pull requests with the new API.

**Comment by artem-sidorenko** [comment]
I’m comaintainer in some repos, where approval is required in order to merge the PR. Without this feature this app is almost useless for me:-(
